TicketDesk/ngWebClient-StandAlone/src/app/services/ad-contact.service.ts

```diff
@@ -11,9 +11,25 @@ import * as settings from '../app-settings';
 @Injectable()
 export class AdContactService {
 
-  constructor() { }
+  constructor(private http: HttpClient) { }
 
   getContactCardInfo(userID: string): Observable<UserDetails> {
-    return Observable.of({firstName: 'Juan', lastName: 'Perez', phoneNumber: '(503)555-1515', email: 'jjpe@wherever.com', userId: userID});
+    return Observable.of({
+      firstName: '<First Name>',
+      lastName: '<Last Name>',
+      phoneNumber: '<Phone Number>',
+      email: 'stillneed@activedirectory.com',
+      userId: userID});
+  }
+
+  private handleError(error: HttpErrorResponse): ErrorObservable {
+    if (error.error instanceof ErrorEvent) {
+      // ... this is a client side error, handle it!
```

Comment:
I don't get how general this is supposed to be. Is it going to handle all different kinds of errors? Actually, looking at it again, I guess it's pretty much the sample code. No changes needed, but probably worth talking eventually about where different responses for different problems will be differentiated.

Reply:
yeah it needs to be continued in a new branch for an error handler class that can handle all of our services